app/controllers/orphans_controller.rb

```diff
@@ -7,10 +7,9 @@ def index
 
     @current_sort_column = valid_sort_column
     @current_sort_direction = valid_sort_direction
-
     @filters = filters_params
-    @orphans_before_paginate = Orphan.with_filter_fields.filter(@filters).
-      order(@current_sort_column.to_s + " " +  @current_sort_direction.to_s)
+    @orphans_before_paginate = Orphan.with_filter_fields.filter(@filters)
+      .order(@current_sort_column.to_s + " " +  @current_sort_direction.to_s)
```

Comment:
In Ruby, I feel it's better to have these at the end.  You can't copy & paste this into rails console now.  Why the change?  Not trying to bikeshed; just curious